src/shared/api/base.ts

```diff
@@ -0,0 +1,75 @@
+import { API_URL } from "~shared/config";
+import axios, { AxiosInstance, AxiosResponse, AxiosRequestConfig } from "axios";
+
+class ApiInstance {
```

Comment:
for what is it need?

```ts
export const request = axios.create({
      baseURL: API_URL,
      headers: {
        "Content-Type": "application/json",
      },
    });
```

this should be enough

Reply:
Done in next PR